### ISSUE TITLE
Progress Bars should have an `aria-label` option

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-react-bootstrap#91]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-react-bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-react-bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

RE: https://modus-react-bootstrap.trimble.com/components/progress-bars/

It'd be great if Progress Bars allowed developer to set an `aria-label`. I added this to Modus Bootstrap site recently: https://modus-bootstrap.trimble.com/components/progress-bars/

### Motivation and context

Without `aria-label` on progress bars accessibility tests will fail.
![image](https://user-images.githubusercontent.com/1212885/178942122-475314c7-f854-461f-92aa-87e76e6e9572.png)

REF: https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=axeAPI
